packages/cli/openapi-ir-to-fern/src/buildTypeDeclaration.ts

```diff
@@ -107,7 +107,17 @@ export function buildObjectTypeDeclaration({
             context,
             fileContainingReference: declarationFile
         });
-        if (property.audiences.length > 0) {
+        if (property.audiences.length > 0 && property.nameOverride != null) {
```

Comment:
nit: We might be able to reduce nesting here if we resolve the property name and audiences outside of the condition, then set the value in the `properties` map from there.

Not a big deal either way though, so not blocking.